docs/08_cli.md

```diff
@@ -153,6 +153,14 @@ cadence wf start -tl helloWorldGroup -wt main.Workflow -et 60 -i '"cadence"' -me
 ./cadence workflow list -m
 ```
 
+Use **--query** to list workflows with SQL like query:
+
+```
+./cadence workflow list --query "WorkflowType='main.SampleParentWorkflow' AND CloseTime = missing "
```

Comment:
What's the guidance on when to use single quotes? Is `missing` an enum value or a string?

Reply:
missing is a special enum not a string. I think either signal quotes or double quotes should work. I use single here because the whole query is already using double. 
I think this is not a complete doc for new visibility CLI. I added a simple one here because batch CLI need to refer to it.  @vancexu  do you have a plan to add more documents for new visibility based on ES?

Reply:
yes, I do have plan to add one on cadence page.